dimagi/utils/couch/migration.py

```diff
@@ -13,7 +13,7 @@ class SyncCouchToSQLMixin(object):
     couch_id, for the _id of the Couch doc. Make sure you index couch_id
     and allow it to be null since it can be initially null. Do not include
     this field in _migration_get_fields().
-    You can override 'couch_id_name' if you need to use a different name.
+    You can override '_migration_couch_id_name' if `couch_id` won't work.
```

Comment:
i think it might be worth clarifying this comment to say that `_migration_couch_id_name` lives only on the sync sql to couch class, and not the sync couch to sql class